client/client.go

```diff
@@ -76,16 +78,22 @@ type Client struct {
 	staticPubKey        glow.PublicKey
 	staticPrivKey       glow.PrivateKey
 
+	// Energy multiplier
+	energyMultiplier float64
+	energyDivider    float64
+
 	// Sync primitives.
-	mu     sync.Mutex
-	tg     threadgroup.ThreadGroup
+	mu sync.Mutex
+	tg threadgroup.ThreadGroup
 }
 
 // NewClient will return a new client that is running smoothly.
 func NewClient(baseDir string) (*Client, error) {
 	// Create an empty client.
 	c := &Client{
-		staticBaseDir: baseDir,
+		staticBaseDir:    baseDir,
+		energyMultiplier: EnergyMultiplierDefault,
+		energyDivider:    EnergyDividerDefault,
```

Comment:
Rather than setting the defaults here, I think the defaults should be set in the logic that loads the settings file. That keeps all of the relevant logic in one place.